types/danish-ssn/tslint.json

```
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false,
+        "dt-header": false
+    }
+}
```

```suggestion
{
    "extends": "@definitelytyped/dtslint/dt.json"
}
```
as in comments earlier, no longer required